frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -224,6 +224,12 @@ const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
     }
   };
 
+  const add = () => {
+    if (state.numVolumes < 20) {
```

Comment:
```suggestion
    if (state.numVolumes =< 20) {
```
as per the bugzilla description

Reply:
Also what we are doing if `numVolumes` is greater than 20? I don't think that backend will throw error as the call itself will not be made, if I understand correctly.

Reply:
No you cant reach 21.

Reply:
as per the BZ description

Reply:
Yeah, the condition says, increment the volume by 1 till the volume is less than 20. Therefore, the last value of volume that can be incremented is at 19, to make it 20.

Reply:
So basically here we are restricting till 19, so that it can be incremented to 20. Got it. @a2batic Thanks!